doc/usage.md

```diff
@@ -168,7 +168,7 @@ Every `Error` object we pass back has the properties:
 For example, sending message to a non-existent user:
 
 ```js
-bot.sendMessage(nonExistentUserId, 'text').catch(error => {
```

Comment:
Well, as long as the end-result is you using ES6, go for it.

(ps: not familiar with the IDE)